modules/database/variables.tf

```
@@ -0,0 +1,83 @@
+#  ╺┳┓┏━┓╺┳╸┏━┓┏┓ ┏━┓┏━┓┏━╸
+#   ┃┃┣━┫ ┃ ┣━┫┣┻┓┣━┫┗━┓┣╸
+#  ╺┻┛╹ ╹ ╹ ╹ ╹┗━┛╹ ╹┗━┛┗━╸
+
+variable "provider_region" {
+  type        = string
+  description = "Region for deployment in GCP"
+}
+
+variable "project_id" {
+  type        = string
+  description = "The project to deploy to."
+}
+
+variable "vpc_id" {
+  type        = string
+  default     = ""
+  description = "Provide ID of existing VPC if you want to omit creation of new one"
+}
+
+variable "azs" {
+  type        = list(any)
+  description = "Availability zones to deploy into"
+}
+
+variable "private_network" {
+  type        = string
+  description = "Private network to use"
+}
+
+variable "deployment_name" {
+  type        = string
+  description = "Name of the current deployment."
+}
+
+variable "postgres_username" {
+  type    = string
+  default = "datafold"
+}
+
+variable "postgres_instance" {
+  type = string
+  # https://cloud.google.com/sql/docs/mysql/instance-settings
+  #  Error 400: Invalid request: Only custom machine instance type and shared-core
+  # instance type allowed for PostgreSQL database., invalid
```

```suggestion
  # https://cloud.google.com/sql/docs/mysql/instance-settings
  #  Error 400: Invalid request: Only _custom machine instance_ type and _shared-core_
  # instance type allowed for PostgreSQL database.
```

Is this comment useful for debugging or can we drop it altogether?